Makefile

```diff
@@ -15,7 +15,6 @@ audit:
 
 .PHONY: lint
 lint:
-	golangci-lint run ./... --timeout 10m --skip-dirs=features --verbose --enable=gosec --enable=gocritic --enable=gofmt --enable=gocyclo --enable=bodyclose --enable=gocognit --enable=goimports
```

Comment:
why are we taking linting out of CI?

Reply:
It was causing PRs to fail - Not from failing liniting but failing to download the golangci-lint itself.. And it was never discussed or agreed that it should be added in the first place. None of the other services have any linting running in CI.